src/Components/NavBar.js

```diff
@@ -62,6 +64,38 @@ const NavBar = () => {
         </li>
       </ul>
     </div>
+
+    <div className='responsive'>
+      <ul className="navBar-list">
+          <li className="navElement">
+            <div className="btn-container">
+              <Link className="navLink" to="/">
+                Home
+              </Link>
+            </div>
+          </li>
+          <li>
+              <input type="checkbox" id="menu"/>
+              <label id='collapsible-btn' for="menu"><FontAwesomeIcon icon="gem" /></label>
```

Comment:
```suggestion
              <label id='collapsible-btn' htmlFor="menu"><FontAwesomeIcon icon="gem" /></label>
```
This is the last error that is showing in the console. You need to use htmlFor instead of for.

Reply:
Done 😄